README.md

```diff
@@ -13,6 +13,7 @@ adhesion        | Testing print bed adhesion features, such as brim, skirt and r
 basic           | Generic shapes, mostly geometric. These models may be used to test simple stuff when you just need a bit of infill, or to test front-end features such as linear transformations.
 bridging        | Models featuring lots of overhangs. The overhangs may get bridged properly if the slicer does a good job to orient the skin lines above the bridge properly.
 broken          | Model files that contain faults. They may have syntax mistakes, be corrupted or have non-manifold surfaces. They may not be watertight or have extra geometry inside them. This is intended to test the robustness of the slicer.
+calibration     | Calibration test part.
```

Comment:
Calibration tests are not necessary to test a slicer. I consider this out of scope.